cluster/addons/fluentd-gcp/event-exporter.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: event-exporter-sa
+  namespace: kube-system
+  labels:
+    k8s-app: event-exporter
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+---
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: event-exporter-rb
+  namespace: kube-system
+  labels:
+    k8s-app: event-exporter
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- apiVersion: v1
+  kind: ServiceAccount
+  name: event-exporter-sa
+  namespace: kube-system
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: event-exporter-deployment
+  namespace: kube-system
+  labels:
+    k8s-app: event-exporter
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: event-exporter
+    spec:
+      serviceAccountName: event-exporter-sa
+      containers:
+      - name: event-exporter
+        image: gcr.io/google-containers/event-exporter:v0.1.0
+        command:
+        - '/event-exporter'
+        resources:
+          limits:
```

Comment:
@gmarek Great, thanks. I've observed that the db size when event exporter crashed was ~ the same size as the limit